ocs_ci/deployment/provider_client/storage_client_deployment.py

```diff
@@ -124,6 +122,31 @@ def provider_and_native_client_installation(
         wait_for_machineconfigpool_status(node_type="all")
         log.info("All the nodes are upgraded")
 
+        # Mark master nodes schedulable if mark_masters_schedulable: True
+        if config.ENV_DATA.get("mark_masters_schedulable", False):
+            path = "/spec/mastersSchedulable"
+            params = f"""[{{"op": "replace", "path": "{path}", "value": true}}]"""
+            self.scheduler_obj.patch(params=params, format_type="json"), (
+                "Failed to run patch command to update control nodes as scheduleable"
+            )
```

Comment:
There seems to be missing `assert` on this line, right?
```suggestion
            assert self.scheduler_obj.patch(params=params, format_type="json"), (
                "Failed to run patch command to update control nodes as scheduleable"
            )
```

Reply:
yes I missed the assert, have added in latest commit.